android/src/main/java/com/guichaguri/trackplayer/module/MusicModule.java

```diff
@@ -241,6 +243,33 @@ public void remove(ReadableArray tracks, final Promise callback) {
         });
     }
 
+    @ReactMethod
+    public void updateMetadata(String id, ReadableMap map, final Promise callback) {
```

Comment:
let's rename this `updateMetadataForTrack`

Reply:
Sure

Reply:
I'm about to finish the iOS part so if you could rename them now, we can merge in a few min :)